src/apollo/queries.js

```diff
@@ -444,9 +444,9 @@ export const DASHBOARD_COMULATIVE_DATA = gql`
   }
 `;
 
-export const DASHBOARD_TRANSACTION_HISTORY = gql`
-  query transactions($lastId: ID!, $startTime: Int!) {
-    transactions(first: 1000, where: { id_gt: $lastId, timestamp_gt: $startTime }, orderBy: id, orderDirection: asc) {
+export const DASHBOARD_SWAPS_HISTORY = gql`
+  query swaps($lastId: ID!, $startTime: Int!) {
+    swaps(first: 1000, where: { id_gt: $lastId, timestamp_gt: $startTime }, orderBy: id, orderDirection: asc) {
       id
       timestamp
     }
```

Comment:
```suggestion
export const DASHBOARD_SWAPS_HISTORY_WITH_TIMESTAMP = gql`
  query swaps($lastId: ID!, $startTime: Int!) {
    swaps(first: 1000, where: { id_gt: $lastId, timestamp_gt: $startTime }, orderBy: id, orderDirection: asc) {
      id
      timestamp
    }
    
export const DASHBOARD_SWAPS_HISTORY = gql`
  query swaps($lastId: ID!, $startTime: Int!) {
    swaps(first: 1000, where: { id_gt: $lastId, timestamp_gt: $startTime }) {
      id
    }
```

Since its just the count, order doesn't matter also the return payload of just id is enough for the 24 status. I think graph only needs order and timestamp.

So can split the query into two, so I feel the  `DASHBOARD_SWAPS_HISTORY` will be little faster.

Reply:
Also now that I look at it we can remove the `orderBy: id, orderDirection: asc`, since the data are sorted by `id` by default